File: department-service/src/main/java/pl/piomin/services/department/model/Employee.java
package pl.piomin.services.department.model;

import io.micronaut.serde.annotation.Serdeable;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@ToString
@NoArgsConstructor
@Serdeable
public class Employee {

	private Long id;
	private String name;
	private int age;
	private String position;
	
	public Employee(String name, int age, String position) {
		this.name = name;
		this.age = age;
		this.position = position;
	}

}

File: employee-service/src/main/kotlin/pl/piomin/services/employee/model/Employee.kt
package pl.piomin.services.employee.model;

import javax.persistence.*

@Entity
data class Employee(@Id @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "department_id_seq") @SequenceGenerator(name = "department_id_seq", sequenceName = "department_id_seq") var id: Long,
                    var organizationId: Long, var departmentId: Long, var name: String, var age: Int, var position: String)